### PR TITLE
novi lekse

### PR DESCRIPTION
**Pull request by jkunimune15**
novi lekse da yi suje da redite: https://www.reddit.com/r/pandunia/comments/gtzjqx/new_words_for_minecraft_babe_5/

### CODE REVIEW

File: pandunia-lekse.csv [hunk 1]
```diff
@@ -652,7 +658,8 @@ espania|dex|ES||Spain||España|||||||||||||Hispanio|Espanja|Hiszpania
 esperanti||||Esperanto||Esperanto|||||||||||||Esperanto|esperanto|Esperanto
 estadia|||fra:stade, por:estádio, spa:estadio, rus:стадион (stadion), eng:stadium, tur:stadyum, ara:(ʾistād), hin:स्टेडियम (sṭeḍiyam)|stadium (arena)||estadio (arena)|||||||||||||stadiono (areno)|stadioni (areena)|stadion, arena
 estane|mate|Sn|spa:estaño, por:estanho, fra:étain, ita:stagno, eng:tin|tin|étain|estaño|estanho|||锡|珍|||||||||tina|cynk
-estasa||||stand still||pararse (detenerse)||||||||||||||seisahtaa|nie ruszać się
+estasa||||hold (detain)||mantener (detener)|||||||||||||||
+estasu||||stand still||pararse (detenerse)||||||||||||||seisahtaa|nie ruszać się
```
Comment (barumau) [inline pandunia-lekse.csv:662]: saho!

File: pandunia-lekse.csv [hunk 2]
```diff
@@ -1151,6 +1163,7 @@ jeni baxe||||native language (mother tongue)|langue maternelle|lengua materna|l
 jenoden||||birthday||cumpleaños|||||||||||||naskiĝtago|syntymäpäivä|urodziny
 jenoklase||||caste||casta||||||||||||||sääty (kasti)|kasta
 jenotipe||||genotype|génotype|genotipo|genótipo|генотип||基因型|遺伝子型|||||||||perimä (genotyyppi)|genotyp
+jenozamane||||generation (cohort)||generación (cohorte)||поколение|||||||||||||
```
Comment (barumau) [inline pandunia-lekse.csv:1166]: me fikra "jenogrupe" si max boni.

File: pandunia-lekse.csv [hunk 3]
```diff
@@ -2007,6 +2029,9 @@ nici||||lower||inferior||||||||||||||ala-|poniższy
 nida|||eng: need|need|avoir besoin de|necesitar|necessitar|нуждаться||需要|||cần|ज़रूरत होना||perlu|kuhitaji||bezoni|tarvita|potrzebować
 nide||||need|besoin|necesidad|necessidade|нужда|ضَرُورَة‎||必要|필요||ज़रूरत||keperluan|haja||||potrzeba
 nidi||||necessary (needed)|nécessaire|necesario|necessário|нужный|ضَرُورِيّ‎|必要的|必要|필요하다||ज़रूरी||||||tarpeellinen|potrzebny
+nidoxina||||trust (rely)|fier|confiar (contar con)|confiar (contar com)|доверять (понадеяться)||信赖|信頼する||||||||||
+nidoxinabli||||loyal (faithful, trustworthy)||leal (fiel)||верный||忠实|忠実||||||||||
+nidoxinablia||||loyalty (faithfulness)||lealtad (fidelidad)||верность|||||||||||||
```
Comment (barumau) [inline pandunia-lekse.csv:2034]: boni ideye!

File: pandunia-lekse.csv [hunk 4]
```diff
@@ -1782,6 +1802,7 @@ melovespe||||honeybee||abeja||||||||||||||mehiläinen|pszczoła miodna
 mema||||remember (recall)|se rappeler (se remémorer)|acordarse|lembrar-se (recordar-se)|помнить|تَذَكَّرَ|记得 (回忆)|思い出す|기억하다|nhớ|याद करना|মনে রাখা|ingat||hatırlamak|memori|muistaa|pamiętać
 memba||||belong (be a member)||pertenecer (ser miembro de)||||||||||||||kuulua (olla jäsenenä)|należeć (być członkiem)
 member||||member (component)||miembro (componente)|||||||||||||membro|jäsen (osatekijä, komponentti)|członek
+membrane|||eng:fra:membrane, spa:por:ita:membrana, deu:Membran, rus:мембрана (membrana), may:membran|membrane (film)|membrane|membrana|membrana|плева (перепонка)||膜|膜||||||||||
```
Comment (barumau) [inline pandunia-lekse.csv:1805]: eska mome xudu muta mene da "filme"?

File: pandunia-lekse.csv [hunk 5]
```diff
@@ -2599,6 +2632,7 @@ senegalia|dex|SN||Senegal||Senegal||||||||||||||Senegal|Senegal
 sensa||||sense (detect)||sentir (detectar)|||||||||||||||
 sense||||sense||sentido|||||||||||||||zmysł
 senser||||sensor (detector)|||||||||||||||||
+sensia||||sensation (feeling)|||||||||||||||||
```
Comment (barumau) [inline pandunia-lekse.csv:2635]: me fikra "sense" si "a sensation" e "sensia" si "the senses".

File: pandunia-lekse.csv [hunk 6]
```diff
@@ -29,6 +29,8 @@ aga|||fas:urd:اگر‎‎ (agar), hin:अगर (agar), tur:eğer, + yue:假 (g
 agrese|||eng:deu:fra:agression, pol:agresja, rus:агрессия (agressiya), spa:agresión, por:agressão, may:agresif|attack (assault, aggression)|agression (attaque)|ataque|ataque|нападение (атака)||攻击||||आक्रमण||menarik||saldırı|atako (agreso)|hyökkäys (aggressio)|atak (agresja, napaść)
 agresi||||aggressive||agresivo|||||||||||||agresema|hyökkäävä (aggressiivinen)|agresywny
 aha|||rus:ага (aha), kor: 아하 (aha), eng: uh-huh, pol:aha|understand (realize)|comprendre (réaliser)|entender (comprender)|||||||||||||kompreni|oivaltaa (tajuta)|rozumieć, pojmować
+ahota|||hin:आखेट (ākheṭ), ben:খোজা (khōjā), eng:hunt, rus:охота (ohota), eng:hunt|hunt|chasser|cazar|caçar|охотиться||打猎|狩る||||||||||
```
Comment (barumau) [inline pandunia-lekse.csv:32]: me no suku yi cuze. asli lekse no forma veri grupe.
me wana uza sami lekse pa "hunting (try to catch game)" e "fishing (try to catch fish)". me fikra arabi lekse "s-y-d" abla funcu. vida https://en.wiktionary.org/wiki/%D8%B5%D8%A7%D8%AF#Arabic
hindi, urdi e farsi uza lekse "said" o "seyd".

File: pandunia-lekse.csv (hunk 2 above)
Comment (jkunimune15) [reply pandunia-lekse.csv:1166]: me samoraya. we maxo razonu.

File: pandunia-lekse.csv (hunk 4 above)
Comment (jkunimune15) [reply pandunia-lekse.csv:1805]: sa multi bax, "filme" mena "kinografe" e no mena "faini warke". we saba, me teroru le abla galto ahu.
lekino, ablo we no mal. "film" e "movie" si samonam sa engli e espani, e mome no konfuzu lole. duli lekse da mena "movie" okey.

File: pandunia-lekse.csv (hunk 5 above)
Comment (jkunimune15) [reply pandunia-lekse.csv:2635]: le unisa "sense" e "sensation". we okey?

File: pandunia-lekse.csv (hunk 6 above)
Comment (jkunimune15) [reply pandunia-lekse.csv:32]: vero? me no abla eureka "said" o "seyd" sa mi hindi leksoliste o sa gugol tarjer. aga sahi, me suku "saida" visa "ahota".

**Pull request merged**